app/config/default.json

```diff
@@ -0,0 +1,6 @@
+{
+  "port": 6000,
+  "db": {
+    "url": "postgres://stfwrlrj:up7UlBBX50iObmzLK_QWu1t2zl-nm1YN@kandula.db.elephantsql.com:5432/stfwrlrj"
```

Comment:
credentials shouldn't be stored in the repo. Good way store them in the file that in .gitigore

Reply:
Done